main-packages-list.ocp

```diff
@@ -13,18 +13,19 @@ ipxe-roms-qemu
 iscsi-initiator-utils
 mariadb-server
 mod_ssl
-openstack-ironic-api >= 1:18.3.0-0.20211206200515.0ff3da2.el8
-openstack-ironic-conductor >= 1:18.3.0-0.20211206200515.0ff3da2.el8
-openstack-ironic-inspector >= 10.7.1-0.20210722154052.edf655c.el8
+openstack-ironic-api >= 1:19.0.1-0.20211210165034.c86ca20.el8
+openstack-ironic-conductor >= 1:19.0.1-0.20211210165034.c86ca20.el8
+openstack-ironic-inspector >= 10.9.1-0.20211209161020.21209f3.el8
 parted
 psmisc
+python3-cliff >= 3.9.0-0.20211020191737.734bc0c.el8
```

Comment:
I know it's useful for testing, but we shouldn't have cliff in our packages list.

Reply:
good point, this was actually for testing purpose only, I have to remove it